src/utils/formatters.ts

```diff
@@ -2,12 +2,8 @@ import { BigNumberish, type BigNumber } from 'ethers'
 import { formatUnits, parseUnits } from 'ethers/lib/utils'
 import { formatAmount } from './formatNumber'
 
-export const toDecimals = (value: BigNumberish, decimals?: number | string): number => {
-  return Number(formatUnits(value, decimals))
-}
-
```

Comment:
This was an unnnecessary wrapper as we only once need convert the formatted numbers to numbers.